### DISCUSSION TITLE
shift+tab doesn't wokr in insert mode

### DISCUSSION THREAD

**Discussion by zwyyy456** [LazyVim/LazyVim#551] [Q&A]
I want to use `shfit+tab` to reduce indentation in insert mode, since `shiftround=true` is set. But it doesn't work. And I tried `:map`, it shows as follows:
<img width="1164" alt="图片" src="https://user-images.githubusercontent.com/45388797/230713497-6b9f2b72-8835-4245-ba9a-43918d067c03.png">

Can someone tell me how to enable `shift+tab` for reducing indentation in insert mode?

**Comment by folke** [comment]
`<s-tab>` is a default key binding for luasnip to jump to the previous insert node
